openlabcmd/openlabcmd/plugins/recover.py

```diff
@@ -45,4 +48,21 @@ class Recover(Enum):
                    "--network openlab-net --subnet-range=192.168.0.0/24",
         "reason": "- Subnet cidr: 192.168.0.0/24 is not found.",
     },
+    Recover.ROUTER: {
```

Comment:
As mentioned before, does the router depends on the subnet or network? if yes, maybe we should append the router in network plugin, or introduce some priority mechanism in labchecker in future?

Reply:
Router creation doesn't depend on those resources, but our process flow does. So I move the code into network plugin.